libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/ImageUtils.java

```diff
@@ -875,4 +872,22 @@ public static Bitmap getThumbnail(ContentResolver contentResolver, long id, int
 
         return null;
     }
+
+    private static float getScaleImageBy(float maxSize, Bitmap bmpResized, boolean isPortrait) {
```

Comment:
Is the parameter `isPortrait` necessary here? It seems that we can get height and width dimensions from `bmpResized` parameter.